components/shared/workCard/helper.ts

```diff
@@ -0,0 +1,46 @@
+import { GeneralMaterialTypeCode } from "./../../../lib/graphql/generated/fbi/graphql"
+import { SearchWithPaginationQuery } from "@/lib/graphql/generated/fbi/graphql"
+
+export const displayCreators = (
+  creators: SearchWithPaginationQuery["search"]["works"][0]["creators"],
+  amount: number
+) => {
+  return creators.reduce((acc, creator, index) => {
+    // We shorten to max
+    if (index === amount) {
+      return acc + ", et. al"
+    }
+    if (index > amount) {
+      return acc
+    }
+    return acc + (index > 0 ? ", " : "") + creator.display
+  }, "")
+}
```

Comment:
It is very important that this is consistent with how it has been done in dpl-react. Please double-check it. If you need a pointer to the place in the repository let me know.

Reply:
It is the same, but dynamic in how many we want to shorten it to. In this case I chose two, so that the search results look good + children probably wouldn't be able to soak in more information than two names.

Reply:
Check. Would be nice with a small test here.
That is super quick to implement a unit test for it.